main/src/com/google/refine/ProjectMetadata.java

```diff
@@ -179,10 +227,28 @@ static public ProjectMetadata loadFromJSON(JSONObject obj) {
                     }
                 }
             } catch (JSONException e) {
-                // ignore
+                logger.error(ExceptionUtils.getFullStackTrace(e));
             }
         }
-
+        
+        if (obj.has("importOptionMetaData") && !obj.isNull("importOptionMetaData")) {
```

Comment:
I would spell this `Metadata` rather than `MetaData` to keep it consistent with the rest (and common usage).

Reply:
Agreed. just changed